src/clock-weather-card.ts

```diff
@@ -198,11 +200,11 @@ export class ClockWeatherCard extends LitElement {
     const maxTemp = Math.round(max(maxTemps));
     
     const gradientRange = this.gradientRange(minTemp, maxTemp, temperatueUnit);
-    return dailyForecasts.map((forecast) => safeRender(() => this.renderForecastDay(forecast, gradientRange, minTemp, maxTemp, currentTemp)));
+    return forecasts.map((forecast) => safeRender(() => this.renderForecastItem(forecast, gradientRange, minTemp, maxTemp, currentTemp, hourly)));
   }
 
-  private renderForecastDay(forecast: MergedWeatherForecast, gradientRange: Rgb[], minTemp: number, maxTemp: number, currentTemp: number | null): TemplateResult {
-    const dayText = this.localize(`day.${new Date(forecast.datetime).getDay()}`);
+  private renderForecastItem(forecast: MergedWeatherForecast, gradientRange: Rgb[], minTemp: number, maxTemp: number, currentTemp: number | null, hourly: boolean): TemplateResult {
+    const displayText = !hourly ? this.localize(`day.${new Date(forecast.datetime).getDay()}`) : new Date(forecast.datetime).toLocaleTimeString().substring(0,5);
```

Comment:
this does not take into account a customisable `time_format`. if a user defines `time_format: 12` in the config, the time should be displayed as e.g. `12:00 AM`

Reply:
makes sense. I have overlooked that setting. will add it